EIPS/eip-99999.md

```diff
@@ -0,0 +1,119 @@
+---
+eip: 99999
```

Comment:
```suggestion
eip: 8015
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.